test/images/sample-apiserver/VERSION

```diff
@@ -0,0 +1 @@
+1.7v3
```

Comment:
nit: `1.7.3` or `1.7.0-3`?

Reply:
the previous image was `1.7v2`, so i just bumped it from 2 to 3.

Reply:
Since this is a brand new image, i am just going to start from 1.0